packages/swift/Package.swift

```diff
@@ -0,0 +1,14 @@
+// swift-tools-version:5.4
+import PackageDescription
+
+let package = Package(
+    name: "MXPostMessageDefinitions",
```

Comment:
I _may_ change this name, it depends on how this package is imported/exported by the iOS SDK.